tests/beaker_tests/cisco_interface/routedintf_provider_negatives.rb

```diff
@@ -526,6 +526,174 @@
     logger.info("Check interface instance absence on agent :: #{result}")
   end
 
+  # @step [Step] Requests manifest from the master server to the agent.
```

Comment:
Yes, I am following the existing "negative" cases and they are coded the same way. "defaults" and "non-defaults" are created by loading manifests consisting of all configs.